CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## `3.5.0`
 - Bugfix: Schema validation error is not properly formatted for enumerate type. [(#562)](https://github.com/zowe/zowe-common-c/pull/562)
+- Bugfix: fix a typo in the cross-memory server's help text (#565)
```

Comment:
Add link here, for some reason it is not transformed to URL.

Reply:
Yes, weird. I thought it would implicitly do that.. Well, maybe at some point, we should go through all of them and add the actual links.